definitions/variable/tag_electricity-source.yaml

```
@@ -0,0 +1,23 @@
+- Electricity Source:
+    - Gas|{Turbine Type}|w/ CCS:
+        description: combined or open cycle gas turbine with carbon capture and storage
+    - Gas|{Turbine Type}|w/o CCS:
+        description: combined or open cycle gas turbine without carbon capture and storage
+    - Hydro|Reservoir:
+        description: hydroelectric power plants with reservoirs
+    - Hydro|Run of River:
+        description: hydroelectric power plants without reservoirs
+    - Coal|Lignite|w/ CCS:          
+        description: coal (lignite) power plants without carbon capture and storage    
+    - Coal|Lignite|w/o CCS:          
+        description: coal (lignite) power plants without carbon capture and storage
+    - Waste:
+        description: waste-to-energy plants
+    - Coal|Hard Coal|w/ CCS:
+        description: coal power plants with carbon capture and storage
+    - Coal|Hard Coal|w/o CCS:
+        description: coal power plants without carbon capture and storage
+    - Electricity storage:
```

Always capitalize words in the variable name.
```suggestion
    - Electricity Storage:
```